governance/polls/PPG - Open Market Committee Proposal - July 2, 2022.md

```diff
@@ -0,0 +1,75 @@
+---
+title: PPG - Open Market Committee Proposal - July 2, 2022
+summary: Signal your support or opposition to included parameter changes based on the recommendation of the Maker Open Market Committee.
+discussion_link: [$discussion_link](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-30-june-2022/16248)
+vote_type: Plurality Voting
+categories:
+   - Risk Variable
+   - Rates
+   - High Impact
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-07-04T16:00:00
+end_date: 2022-07-07T16:00:00
+---
+# PPG - Open Market Committee Proposal - July 2, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the [MakerDAO Open Market Committee](https://forum.makerdao.com/t/parameter-proposal-group-makerdao-open-market-committee/7355). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, July 2 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+MKR Token Holders may vote in this poll to express support or opposition to the following parameter changes:
+
+**[Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) changes**
+
+- Increase the **GUNIV3DAIUSDC1-A** Stability Fee to 0.01%.
+- Increase the **UNIV2DAIUSDC-A** Stability Fee to 0.01%.
+- Increase the **WSTETH-A** Stability fee to 2.5%.
+- Reduce the **USDC-A** Stability Fee to 0%.
+- Reduce the **UUSDP-A** Stability Fee to 0%.
+
+**[Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) changes**
+
+- Increase the **UNIV2DAIUSDC-A** Maximum Debt Ceiling to 300 million.
+- Reduce the **LINK-A** Maximum Debt Ceiling to 25 million.
+- Reduce the **WSTETH-A** Maximum Debt Ceiling to 150 million.
+- Reduce the **WSTETH-B** Maximum Debt Ceiling to 100 million.
+- Reduce the **YFI-A** Maximum Debt Ceiling to 25 million.
+- Reduce the **MATIC-A** Maximum Debt Ceiling to 20 million.
+
+**[Target Available Debt](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) changes**
+
+- Increase the **UNIV2DAIUSDC-A** Target Available Debt (`gap`) to 20 million. 
```

Comment:
units please, would put parameter name once in the sub-heading and leave it out of the individual parameter changes

```suggestion
- Increase the **UNIV2DAIUSDC-A** Target Available Debt to 20 million DAI. 
```